### ISSUE TITLE
Add initial test

### ISSUE THREAD

**Issue by schaefi** [OSInside/kiwi-functional-tests#2]
First steps to use the openQA system

**Comment by schaefi** [comment]
I'm just putting my notes here:

Install openQA-client package from here

```
$ zypper ar https://download.opensuse.org/repositories/devel:/openQA/openSUSE_Leap_15.1 openqa
$ zypper ar https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.1/openSUSE_Leap_15.1 openqa-deps

$ zypper in openQA-client
```

Setup openQA account and credentials config in

```
~/.config/openqa/client.conf 

[openqa.opensuse.org]
key = ...
secret = ...
```

Schedule a job for an image for this PR

```
openqa-cli api --o3 -X post jobs \
  
 ISO_1_URL=https://download.opensuse.org/repositories/Virtualization:/Appliances:/Images:/Testing_x86:/suse/images/iso/openSUSE-Tumbleweed.x86_64.iso \
  _GROUP=0 \
  TEST=mschaefer_appliance_test \
  BUILD=X ARCH=x86_64 \
  CASEDIR=https://github.com/OSInside/kiwi-functional-tests/pull/1#test-simple-boot

{"id":1430683}

https://openqa.opensuse.org/tests/1430683
```

**Comment by schaefi** [comment]
https://openqa.opensuse.org/tests/1430717

**Comment by schaefi** [comment]
For images that comes compressed as ".raw.xz" a decompression is require prior use. This can be achieved with:

```
HDD_1_DECOMPRESS_URL
```

instead of 

```
HDD_1_URL
```

**Comment by dcermak** [comment]
Initial draft has been merged via #10.
